src/app/api/common/delegates/getDelegates.ts

```diff
@@ -609,5 +609,5 @@ async function getVoterStats(
 }
 
 export const fetchDelegates = cache(getDelegates);
-export const fetchDelegate = cache(getDelegate);
-export const fetchVoterStats = cache(getVoterStats);
```

Comment:
it's fine to leave `cache` here -- I think it's still valuable for deduping in case there's an area that we don't use data-cache for. It's a react only thing